app/controllers/authenticate_controller.rb

```diff
@@ -16,7 +18,10 @@ def authenticate_okta
       request_ip: request.ip
     )
 
+    render_authn_token(auth_token)
     Rails.logger.debug("AuthenticateController#authenticate_okta - authentication token: #{auth_token.inspect}")
+  rescue => e
+    handle_oidc_authentication_error(e)
```

Comment:
OIDC specific failures should be handled in the Handler.  I know this runs counter, but it allows us to isolate OIDC exceptions to the OIDC context.